src/containers/layouts/messaging/chat1/chat1.container.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { NavigationScreenProps } from 'react-navigation';
+import { Chat1 } from './chat1.component';
+import {
+  Conversation,
+  Message,
+  Profile,
+} from '@src/core/model';
+import { conversation5 } from '@src/core/data/conversation';
+import {
+  profile1,
+  profile2,
+} from '@src/core/data/profile';
+
+interface State {
+  newMessageText: string;
+  conversation: Conversation;
+}
+
+export class Chat1Container extends React.Component<NavigationScreenProps, State> {
+
+  public state: State = {
+    newMessageText: '',
+    conversation: conversation5,
+  };
+
+  private onNewMessageChange = (newMessageText: string): void => {
+    this.setState({ newMessageText });
+  };
+
+  private onMessageAddPress = (): void => {
```

Comment:
you should just disable this button until input has no text

Reply:
Refactored